### ISSUE TITLE
Incorrect name is provided for link as "#" available beside Initial setup heading: A11y_Follow a workshop_Initial Setup_Name Role Value.

### ISSUE THREAD

**Issue by AnKushSingh05** [microsoft/moaw#125]
### GitHub Tags
#A11yTCS; #A11ySev2; #ChromiumEdge; #WCAG4.1.2; #Win11; #GH_Maow_Web_Mar24; #Maow; #Name Role Value; #AILimited: #DesktopWeb; #A11yMAS;

### Environment Details:
Application Name: Maow
#URL: https://moaw.dev/workshop/create-workshop/?step=1
OS: Windows 11 Enterprise 23H2 (OS build 22631.3296)
Microsoft Edge: Version 122.0.2365.92 (Official build) (64-bit)

### Repro Steps:

1. Hit the URL https://moaw.dev/workshop/create-workshop/
2. Create A Workshop screen will get open.
3. TAB till "Initial setup" control present in left navigation and press ENTER.
4. TAB till "Initial setup" section.
5. Verify that correct name is provided for link as "#" available beside Initial setup heading or not.
### Actual Result:
Incorrect name is provided for link as "#" available beside Initial setup heading.

### Expected Result:
Correct name should be provided for the link available beside Initial setup heading.

### User Impact:
Screen reader users will get the wrong information if incorrect name is provided.

### Attachment:
![Bug_125](https://github.com/microsoft/moaw/assets/124244478/cd49c4cc-d823-4996-87c6-88d3b789ab0a)

**Comment by sinedied** [comment]
@AnKushSingh05 this one has been fixed, could you validate it?

**Comment by AnKushSingh05** [comment]
Verified the issue in below environment
https://moaw.dev/workshop/create-workshop/?step=1#
Issue is fixed. Hence closing the bug

![Screenshot (1954)](https://github.com/microsoft/moaw/assets/124244478/68c6b09b-edbe-4fab-9348-78f9497384ac)
